### PR TITLE
Changed named to copy all headers to make clear

### PR DESCRIPTION
**Pull request by abist**
Fixes https://github.com/microsoft/vscode-mssql/issues/1405

### CODE REVIEW

File: src/views/htmlcontent/src/js/components/app.component.ts [hunk 1]
```diff
@@ -468,7 +468,7 @@ export class AppComponent implements OnInit, AfterViewChecked {
             case 'copyWithHeaders':
                 this.dataService.copyResults(event.selection, event.batchId, event.resultId, true);
                 break;
-            case 'copyHeaders':
+            case 'copAllyHeaders':
```
Comment (aaomidi) [inline src/views/htmlcontent/src/js/components/app.component.ts]: extra y

Comment (abist) [reply src/views/htmlcontent/src/js/components/app.component.ts]: Good catch!

File: src/views/htmlcontent/src/js/components/app.component.ts [hunk 2]
```diff
@@ -468,7 +468,7 @@ export class AppComponent implements OnInit, AfterViewChecked {
             case 'copyWithHeaders':
                 this.dataService.copyResults(event.selection, event.batchId, event.resultId, true);
                 break;
-            case 'copyHeaders':
+            case 'copAllHeaders':
```
Comment (aaomidi) [inline src/views/htmlcontent/src/js/components/app.component.ts]: Still a typo here :P

Comment (abist) [reply src/views/htmlcontent/src/js/components/app.component.ts]: There must be something wrong with my keyboard aargh

**Pull request merged**